etl/steps/data/garden/demography/2022-11-30/life_expectancy.py

```diff
@@ -342,6 +354,34 @@ def merge_dfs(df_wpp: pd.DataFrame, df_hmd: pd.DataFrame, df_zij: pd.DataFrame,
     return df
 
 
+def add_americas(frame: pd.DataFrame) -> pd.DataFrame:
+    """Estimate value for the Americas using North America and LATAM/Caribbean."""
+    # filter only member countries of the region
+    region_members = ["Northern America", "Latin America and the Caribbean"]
+    df = frame.loc[frame["country"].isin(region_members)].copy()
+    # add population
+    df = add_population_americas(df)
+    # estimate values for regions
+    # y(country) = weight(country) * metric(country)
+    df["life_expectancy_0"] *= df.population
+    df["life_expectancy_15"] *= df.population
+    df["life_expectancy_65"] *= df.population
+    df["life_expectancy_80"] *= df.population
+    # z(region) = sum{ y(country) } for country in region
+    df = df.groupby("year", as_index=False).sum(numeric_only=True)
```

Comment:
Keep in mind that `groupby().sum()` will treat nans as zeros. I don't know if that could bias the weights. It would be good to do some further sanity checks here.

Reply:
added an assert to ensure there is no NaN before doing groupby